web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -1938,6 +1938,11 @@ private BaseQueryResponse next(final String id, boolean checkForContentLookup) {
         RunningQuery query = null;
         Query contentLookupSettings = null;
         try {
+            if (!id.matches("[a-z\\d-]+")) {
+                log.error("Invalid query id: " + id);
```

Comment:
I don't know if the ID endpoint is currently case sensitive.  If it isn't, this change will make it so, which may or may not be the intention.

Reply:
Good point, I'll add upper cases in as well just in case.